test/index.js

```diff
@@ -64,20 +68,20 @@ test('it creates a new instance', ({ truthy }) => {
 test.cb('it blows up when a hook does not exist', ({ end, is, truthy }) => {
   const plugin = new HookShellScriptPlugin(testHooks);
   try {
-    plugin.apply({ options: {}, hooks: {} });
+    plugin.apply({ options: {}, hooks: {}, getInfrastructureLogger: createLogger });
   } catch (e) {
     truthy(e instanceof Error);
-    is(e.message, '\n[HookShellScriptPlugin] The hook beforeRun does not exist on the Webpack compiler.\n');
+    is(e.message, '[HookShellScriptPlugin] The hook beforeRun does not exist on the Webpack compiler.');
```

Comment:
Only the thrown error message is prefixed with `[HookShellScriptPlugin]`, as webpack prepends all other log entries with that string.